app/cdap/components/PipelineTriggers/EnabledTriggersTab/EnabledInlineTriggerRow.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from 'react';
+import IconSVG from 'components/shared/IconSVG';
+import PipelineTriggersStore from 'components/PipelineTriggers/store/PipelineTriggersStore';
+import { getGroupInlinePipelineInfo } from 'components/PipelineTriggers/store/PipelineTriggersActionCreator';
+import LoadingSVG from 'components/shared/LoadingSVG';
+import PayloadConfigModal from 'components/PipelineTriggers/PayloadConfigModal';
+import T from 'i18n-react';
+import NamespaceStore from 'services/NamespaceStore';
+
+const TRIGGER_PREFIX = 'features.PipelineTriggers';
+
+interface IEnabledInlineTriggerRowProps {
+  isExpanded: boolean;
+  trigger: any;
+  groupSchedule: any;
+  loading: boolean;
+  info: any;
+  pipelineName: string;
+  disableError: string;
+}
+
+export default function EnabledInlineTriggerRow(props: IEnabledInlineTriggerRowProps) {
+  const statePipelineName = PipelineTriggersStore.getState().triggers.pipelineName;
+
+  const renderLoading = () => {
+    return (
+      <div className="text-center">
+        <LoadingSVG />
+      </div>
+    );
+  };
+
+  const renderContent = () => {
+    const { trigger, groupSchedule, info, pipelineName } = props;
+
+    const namespace = NamespaceStore.getState().selectedNamespace;
+    const triggeredPipelineInfo = {
+      id: pipelineName,
+      namespace,
+    };
+
+    const triggeringPipelineInfo = {
+      id: trigger.programId.application,
+      namespace: trigger.programId.namespace,
+    };
+
+    const events = trigger.programStatuses;
+    const completed = events.indexOf('COMPLETED') > -1;
+    const killed = events.indexOf('KILLED') > -1;
+    const failed = events.indexOf('FAILED') > -1;
+
+    return (
+      <div className="row-content">
```

Comment:
I know this is probably using previous classes to make it easier but it would be good to eliminate some dependencies on the scss